test2.js

```diff
@@ -0,0 +1,19 @@
+var foo = '' ;
+
+if(foo  === "bar") {
```

Comment:
Codacy has a fix for the issue: [Replace `(foo·` with `·(foo`](https://release.k8s.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=5796)
```suggestion
if (foo === "bar") {
```